PACKAGE.md

```diff
@@ -0,0 +1,88 @@
+# Membrane Package
+
+## Overview
+
+Membrane is a Python-based authentication package that leverages JWT for secure, token-based authentication. It is specifically designed to integrate with Flask applications, providing a streamlined approach for authentication management.
+
+## Installation
+
+```bash
+pip install --no-cache-dir git+https://github.com/ai-cfia/membrane-backend.git
+```
+
+**Note:** The package is still in development. Install from the development branch on `git+https://github.com/ai-cfia/membrane-backend.git@k-allagbe/issue35-packaging-membrane`
+
+## Features
+
+- Simple Flask blueprint integration
+- JWT Token-based authentication
+- Flexible and customizable
+- Error handling for JWT decoding
+
+## Usage
+
+### Basic Setup
+
+Import the `configure` function and call it to set up the package.
+
+```python
+from membrane import configure
+
+# Configuration
+configure(
+    app=flask_app_instance,
+    certificate=path_to_certificate,
+    token_expiration=3600,
+    redirect_path='/',
+    require_login=True
+)
+```
+
+### Adding Login and Logout Routes
+
+If you wish to add login and logout routes to your Flask application, you can simply register the Membrane blueprint:
+
+```python
+from flask import Flask
+from membrane import blueprint as membrane_blueprint
```

Comment:
I think this has been moved to membrane.client.

we probably need to integrate something to extract and test our example code in our github-workflows, such as:

https://github.com/boxed/pytest-readme

Reply:
Done